DuggaSys/tests/courseedservice-test.php

```diff
@@ -0,0 +1,254 @@
+/*
+----------------------------------------------------------
+    Courseedservice.php implement test file
+----------------------------------------------------------
+*/
+ 
+<?php
+ 
+include "../../Shared/test.php";
+ 
+$testsData = array(
+
+    /* Insert into course */
+
+    'Insert into course test' => array(
+        'expected-output' => '{"debug":"NONE!","motd":"UNK"}',
```

Comment:
you need to add the expected output!

it can be found here
![image](https://github.com/HGustavs/LenaSYS/assets/102600391/5aca7d6a-5cf4-4e75-b8db-b432f72ad6f2)
for the correct output to be registered there, you must supply exactly the same values as the values used in the test.  
you will no doubt notice that there might be more values then the ones noted in the document, these are to be added as well (note, the image is not of your actual case)
![image](https://github.com/HGustavs/LenaSYS/assets/102600391/6d1bf934-01a3-4fc2-a388-231efc4ab10e)

Reply:
I might as well point out that there might be some data that needs to be manually removed from the JSON string.